README.md

```diff
@@ -1,40 +1,49 @@
 # How to use the split utility
 
-The split utility is used by giving it a LLVM bitcode file and it splits all of
-the functions in it into separate files. 
+The split utility operates on a LLVM bitcode file and it splits all of
+the functions in it into separate (`.bc`) files.
 
-Obtaining the final LLVM bitcode file can be done in multiple ways. The easiest
-one being to use
-[whole-program-llvm](https://github.com/travitch/whole-program-llvm) that gives
-you the bitcode used to create the executable.
+Obtaining the final "fat" LLVM bitcode file can, theoretically, be done in multiple ways. 
+The recommended way is to use CapableVMs fork of "Go Whole Program LLVM", gllvm.
+For example, to get the bitcode from a generic Makefile project: 
 
-For example to get the bitcode from the cpython executable you build it like
-that: 
+- Install gllvm: 
 ```
-LLVM_COMPILER="clang" CC="wllvm" ./configure 
-LLVM_COMPILER="clang" make
-``` 
+go get github.com/capablevms/gllvm/cmd/...
+```
+
+- Build with GLLVM:
+```
+GCLANG=$HOME/go/bin/gclang
+export GET_BC=$HOME/go/bin/get-bc
+
+CC=$GCLANG LLVM_COMPILER_PATH=path/to/llvm/bin make
+# or CC=$GCLANG ./configure if you are feeling lucky
+```
+
+- Extract the bitcode:
 
-Then extract the bitcode file from the binary using the `extract-bc` utility
-from the `whole-program-llvm` project:
 ```
-extract-bc python 
-``` 
+$GETBC path/to/binary
+```
+
+Finally, the split utility can be launched with:
 
-The last step would be to execute the split utility. This can be done by 
-running it like that: 
 ```
-./split-llvm-extract python.bc -o outdir
+./split-llvm-extract path/to/binary.bc -o outdir
 ```
 
-## Other variants
+To run the version of the binary just split, it is necessary to join the parts (in `outdir`) together as shared libraries.
```

Comment:
Let's wrap these lines to the same length as the others in this file.

Reply:
Fix in [2fe2b9e](https://github.com/capablevms/llvm-function-split/pull/37/commits/2fe2b9e95ef3df217b2dbd2c87f813e8bf6946dd), I *think".

Reply:
Nope, that's made it much worse ;) That commit has totally unwrapped a number of lines.

It looks to me (at a quick glance) that this file, in master, has lines wrapped to 80ish chars. Let's just keep things the same, whatever that is.

Reply:
What about [78da4f5](https://github.com/capablevms/llvm-function-split/pull/37/commits/78da4f5812dab9d8c692a50cd46e21c00130bae9)?

Reply:
OK, but that's reformatted the whole file ;) What I'm really asking is: revert back to https://github.com/capablevms/llvm-function-split/pull/37/commits/085d18545c222b67a5fbdbef4b9824fa44f98869 then wrap only those lines/paragraphs you changed in that commit.

Reply:
Check [57183c8](https://github.com/capablevms/llvm-function-split/pull/37/commits/57183c826455ce015458d5f958ae3fa9c98bef31), please.